python/src/definitions.hpp

```diff
@@ -142,8 +143,20 @@ void addStandardTableDefinitions( PythonClass& table ) {
     "table\n\n"
     "Arguments:\n"
     "    filename    the file name and path"
+  )
+  .def(
+
+    "to_file",
+    [] ( const Table& self, const std::string& filename ) {
+
+      return njoy::NDItk::toFile( self, filename );
+    },
+    "Read an ACE table from a file\n\n"
```

Comment:
NDI

Reply:
copy/paste is the bane of my existence

Reply:
this has been addressed already (it no longer appears in the latest version). Don't know where though.